### ISSUE TITLE
Very Little Documentation

### ISSUE THREAD

**Issue by ElijahAhianyo** [dev-sushi/sushi#29]
Hi there, I think this is a great project. However, there seems to be very limited documentation on how to get started with this and what the functionalities are there. Went through the examples which seem to be in c++ and c (not sure if these are the only supported languages?). 
Long and short, I think it would be pretty useful to have detailed documentation on this .

**Comment by ksawery29** [comment]
@ElijahAhianyo For now I added a few pages [in github wiki](https://github.com/dev-sushi/sushi/wiki). I will finish it later.

**Comment by snacsnoc** [comment]
@ksawery29 I wouldn't mind attacking this issue, how can I contribute? So far I have a Markdown document, would you accept a pull request on the README.md?

**Comment by ksawery29** [comment]
> @ksawery29 I wouldn't mind attacking this issue, how can I contribute? So far I have a Markdown document, would you accept a pull request on the README.md?

What about hosting docs on something like read the docs? It would be better when people want to contribute and generally reading from the documentation would be a lot easier.

**Comment by snacsnoc** [comment]
Yup, that would work fine! Github doesn't accept pull request for the Wiki so that may be the best solution to have a central place for docs.

**Comment by ksawery29** [comment]
@snacsnoc see this https://github.com/dev-sushi/docs

**Comment by ksawery29** [comment]
@snacsnoc As I am moving the documentation I think it might be easier to use some other wiki host (or maybe create our own using mdx) so we don't have to move all documentation back to rst? It would also be easier for new people that want to contribute.

**Comment by snacsnoc** [comment]
@ksawery29 MdDocs seems much better, I would prefer md over rst for the simplicity. Great work

**Comment by ksawery29** [comment]
@ElijahAhianyo Can the issue be closed now?

**Comment by ksawery29** [comment]
Docs will be ready probably tomorrow because I finished today all high priority tasks. If anyone is still interested in contributing pull requests are welcome!
